src/main/kotlin/com/jetbrains/micropython/devices/WemosD1MiniProDeviceProvider.kt

```diff
@@ -0,0 +1,22 @@
+package com.jetbrains.micropython.devices
```

Comment:
Could you please add the license comment here? See [an example](https://github.com/vlasovskikh/intellij-micropython/blob/master/src/main/kotlin/com/jetbrains/micropython/devices/MicroPythonDeviceProvider.kt).